DEPLOYING.md

```diff
@@ -125,3 +174,62 @@ To check the logs for a specific service:
 sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml logs -f ckan
 ```  
 *(exit the logs by pressing Ctrl+C at any time)*
+
+To get inside a running container
+
+```
+sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml exec {service-name} bash
+```  
+
+## Creating the sysadmin user
+
+In order to create organizations and give other user proper permissions, you will need sysamin user(s) who has all the privileges. You can add as many sysadmin as you want. To create sysamin user:
+
+```
+# ssh into ckan container
+sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml exec ckan bash
+
+# Creat sysadmin user using paster CLI tool
+/usr/local/bin/ckan-paster --plugin=ckan sysadmin add {username} --password={password} --email={email} - /etc/ckan/production.ini
+
+# Example
+/usr/local/bin/ckan-paster --plugin=ckan sysadmin add ckan_admin --password=iemae7Ai --email=info@datopian.com - /etc/ckan/production.ini
+```
+
+## Sysadmin Control Panel
+
+Here you can edit portal related configuration, like website title, site logo or add custom styling. Login as sysadmin and navigate to `ckan-admin/config` page and make changes you need. Eg: https://demo.ckan.org/ckan-admin/config
+
+
+## Installing and enabling a new extensions
+
+CKAN allows installing various extensions (plugins) to the existing core setup. In order to enable/disable them you will have to install them and include into the ckan config file.
+
+To install extension you need to modify `POST_INSTALL` section of ckan service in `.docker-compose.vital-strategies-theme.yaml`. Eg to install s3filestore extension
```

Comment:
I commented above about this. It was "referenced before assignment" :stuck_out_tongue:

Reply:
@nightsh I agree about the above but will keep this part as it's an example, ok?

Reply:
It's a good example :)